linkerd/app/core/src/http_tracing.rs

```diff
@@ -1,23 +1,22 @@
 use linkerd_error::Error;
 use linkerd_opencensus::proto::trace::v1 as oc;
-use linkerd_trace_context as trace_context;
-use std::collections::HashMap;
-use std::{error, fmt};
+use linkerd_stack::layer;
+use linkerd_trace_context::{self as trace_context, TraceContext};
+use std::{collections::HashMap, error, fmt, sync::Arc};
 use tokio::sync::mpsc;
 
-const SPAN_KIND_SERVER: i32 = 1;
-const SPAN_KIND_CLIENT: i32 = 2;
+pub type SpanSink = Option<mpsc::Sender<oc::Span>>;
```

Comment:
nit: imo having a type alias with the _same_ name as the trait is a little bit unclear, but maybe that's just me.